src/spaces/grid_multi.jl

```diff
@@ -64,7 +64,9 @@ example for usage of this advanced specification of dimension-dependent distance
 where one dimension is used as a categorical one.
 """
 function GridSpace(
-        d::NTuple{D,Int}; periodic::Bool = true, metric::Symbol = :chebyshev
+        d::NTuple{D,Int};
+        periodic::Union{Bool,SVector{D,Bool},NTuple{D,Bool}} = true,
```

Comment:
```suggestion
        periodic::Union{Bool,SVector{D,Bool},NTuple{D,Bool}} = true,
```

why allowing also a `SVector`? In Gridspaces it seems better just to use tuples

Reply:
I allowed both `SVector` and `NTuple` because the `periodic` argument of `ContinuousSpace` goes through `GridSpace` anyway so thought we might as well allow both.
It's also true that there is no real reason to support `SVector` even for `ContinuousSpace`, since it's just extra keystrokes in this case, but there's no real reason to not support it either. Any iterable with fixed size is the same here.
If preferred we can just limit to `NTuple`s overall.

Reply:
In my opinion since there is not a good reason to support it, I would not support it